pkg/scheduler/plugins/coscheduling/core/gang.go

```diff
@@ -95,6 +95,44 @@ func NewGang(gangName string) *Gang {
 	}
 }
 
+func (gang *Gang) Marshal() *GangMarshal {
```

Comment:
cool, A separate output struct is better,  remove the json tags in the original struct?

Reply:
fixed